pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
+addopts = --setup-show -svv
```

Comment:
This is causing a lot of spam in our unit tests output report. I will prepare a PR to separate the configs